main.js

```diff
@@ -0,0 +1,16 @@
+function daysInYear(year) {
+    try {
+        if (!Number.isInteger(year)) {
+            throw 'not valid';
+        }
+
+        let sum = 0;
+
+        for (let x = 0; x < 12; x++) {
+            sum += new Date(year, x, 0).getDate();
+        }
+        return sum;
+    } catch(e) {

```

Comment:
ok, but the main propose of throwing custom errors is catching them in another function